knotx-knot/knotx-knot-service/src/main/java/io/knotx/knot/service/impl/ServiceKnotProxyImpl.java

```diff
@@ -84,10 +84,7 @@ private KnotContext createSuccessResponse(KnotContext inputContext) {
   }
 
   private Observable<FragmentContext> compileHtmlFragment(Fragment fragment) {
```

Comment:
We should return an `Observable` here because latter processing concerns stream of events (processing all services, more than one is possible).